compliance_checker/cf/cf.py

```diff
@@ -2784,7 +2794,6 @@ def check_climatological_statistics(self, ds):
                 return ret_val
             # handle 1-d and 2d coordinate bounds
             if (clim_var.ndim + 1 != ds.variables[clim_var.climatology].ndim):
-                valid = False
```

Comment:
Yeah ok to remove, I think. I don't see how this variable can be used in any way if this `if` block is executed.